Cargo.toml

```diff
@@ -13,7 +13,8 @@ members = [
 ]
 
 [patch.crates-io]
-halo2_proofs = { git = "https://github.com/appliedzkp/halo2.git", tag = "v2022_02_23" }
+#halo2_proofs = { git = "https://github.com/appliedzkp/halo2.git", tag = "v2022_02_23" }
+halo2_proofs = { git = "https://github.com/scroll-tech/halo2.git", rev = "4604e7b" }
```

Comment:
Once https://github.com/appliedzkp/halo2/pull/38 is merged. I'll release a new tag and let's use it. Revs are really confusing and difficult to follow.